tgui/packages/tgui/styles/main.scss

```
@@ -70,6 +70,7 @@
 @include meta.load-css('./interfaces/AccountsUplinkTerminal.scss');
 @include meta.load-css('./interfaces/MedalsViewer.scss');
 @include meta.load-css('./interfaces/GamePreferences.scss');
+@include meta.load-css('./interfaces/PersonalCrafting.scss');
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@meta.load-css` (scss/load-partial-extension)

(scss/load-partial-extension)